WcaOnRails/config/locales/en.yml

```diff
@@ -305,6 +305,7 @@ en:
         on_the_spot_registration: "On the spot registrations"
         on_the_spot_entry_fee_lowest_denomination: "On the spot base registration fee"
         allow_registration_edits: "Registration edits"
+        allow_registration_self_delete_after_acceptance: "Registration self-deletion"
```

Comment:
"Registration Cancellations"

Reply:
Yep that is indeed a better wording! I will probably make the 'C' lower case since "Registration edits" has the second word lowercase too